election.html

```diff
@@ -8,285 +8,288 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
-        integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+          integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
 
     <link type="text/css" rel="stylesheet" href="style-election.css">
 
     <!-- development version, includes helpful console warnings -->
     <script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
 
     <script src="irma.js" defer></script>
+    <script src="global.js">
+      // --------------------------------
+      // globals loaded from globals file
+      // --------------------------------
+    </script>
     <script type="text/javascript">
-        const AWS_IRMA_SERVER = 'http://13.54.7.159:8088';
-        const IRMA_SERVER = AWS_IRMA_SERVER;
-        const AUTH_METHOD_TOKEN = 'token';
-        const PRIVATE_TOKEN = 'secret-fake-token';
-
-        const MEMBERSHIP_ATTRIBUTES = [
-            [
-                ['inz-demo.internetnz.id.number']
-            ],
-            [
-                ['inz-demo.internetnz.membership.validUntil']
-            ],
-        ]
-
-        function discloseOrSign(attributes, label = '', message = '') {
-            const labelRequest = !label ? {} : { 'labels': { '0': { 'en': label, 'nl': label } } };
-            const request = !message ? {
-                '@context': 'https://irma.app/ld/request/disclosure/v2',
-                'disclose': attributes,
-                ...labelRequest
-            } : {
-                    '@context': 'https://irma.app/ld/request/signature/v2',
-                    'message': message,
-                    'disclose': attributes,
-                    ...labelRequest
-                };
-            return doSession(request).then(function (result) {
-                showSuccess('Success, Your IRMA ID is : <strong>' + result.disclosed[0][0].rawvalue + '</strong>');
-                return getDiscloseResultRawValue(result);
-            });
+      const MEMBERSHIP_ATTRIBUTES = [
+        [
+          [ATTRIBUTE.INTERNETNZ_MEMBERSHIP.ID]
+        ],
+        [
+          [ATTRIBUTE.INTERNETNZ_MEMBERSHIP.VALID_UNTIL]
+        ],
+      ]
+
+      function discloseOrSign(attributes, label = '', message = '') {
```

Comment:
Can we move these functions to global.js as well?

Reply:
We can do it. Right now, it seems that there are slight changes on implementations, then I was thinking about doing that separately not to have too much change on this PR.

Reply:
fair enough!